public/index.html

```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <script src="https://use.fontawesome.com/3f7a2eccd4.js"></script>
```

Comment:
I also think that no modifications should be made to this file @filoxo 
I'll remove this. Let's use the react-icons component for icons.